database/seeders/ProfileImagesSeeder.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\ProfileImage;
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class ProfileImagesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for ($j = 1; $j < 7; $j++) {
+            ProfileImage::create([
+                    'profile_id' => $j,
```

Comment:
For $j logic, that is not good!.
Pls, get profiles table. And then, pass id randomly with faker.